packages/toolpad-core/src/runtime.tsx

```diff
@@ -111,7 +120,7 @@ export function NodeRuntimeWrapper({
             componentConfig,
           }}
         >
-          {children}
+          <ComponentsProvider value={components}>{children}</ComponentsProvider>
```

Comment:
There already is a global provider in the runtime, the one that is used for `useComponents`. My suggestion was to move that one to `@mui/toolpad-core`, so that we can also use `useComponents` inside of `@mui/toolpad-components`.

I don't see why we need to recreate that context inside of every `NodeRuntimeWrapper` if there already is a global contetx with the same value

Reply:
Alright my bad, I'm not sure why initially I got errors complaining about missing provider, but I tried it again and all seems to just work fine now wihtout the need for extra providers 👍